Assets/Scripts/Gamestate/AIManager.cs

```diff
@@ -260,6 +276,55 @@ private void FindPlayers()
 
     }
 
+    private Transform FindClosestPlayer(out float closestDistance)
+    {
+        Transform closestPlayer = null;
+        closestDistance = ignoreAwareRadius;
+
+        var areOnlyAiPlayersLeft = trackedPlayers.All(p => Peer2PeerTransport.PlayerDetails.Any(pd => p.id == pd.id && pd.type is PlayerType.AI));
+
+        foreach (var player in TrackedPlayers)
+        {
+            var targetDirection = player.AiTarget.transform.position - transform.position;
+            var hitDistance = targetDirection.magnitude;
+
+            if (areOnlyAiPlayersLeft)
+            {
+                if (hitDistance > closestDistance)
+                    continue;
+
+                closestDistance = hitDistance;
+                closestPlayer = player.AiTarget.transform;
+                ShootingTarget = player.AiAimSpot;
+                continue;
+            }
```

Comment:
There seems to be a problem with something inside the areOnlyAiPlayersLeft, making it so the AIs can choose to stand still firing at eachother from places they can't hit eachother, thus softlocking the game

Here is an example spot from The Pit
![image](https://github.com/user-attachments/assets/929b9aa3-f4f5-4f3d-9436-320bf594c88d)

Reply:
Fixed. Removed the ShootingTarget and continue; lines, instead setting DestinationTarget.